tests/suites/test_suite_cipher.function

```diff
@@ -589,6 +593,14 @@ void dec_empty_buf( int cipher,
     /* Initialise context */
     cipher_info = mbedtls_cipher_info_from_type( cipher );
     TEST_ASSERT( NULL != cipher_info);
+
+    if( cipher_info->type == MBEDTLS_CIPHER_CHACHA20 ||
+        cipher_info->type == MBEDTLS_CIPHER_CHACHA20_POLY1305 )
+        iv_len = 12;
+
+    ASSERT_ALLOC( iv, iv_len );
```

Comment:
There is no need to dynamically allocate memory for the iv buffer.

Reply:
There's one advantage of only allocating the minimal appropriate size: if the `iv_len` is overwritten by anything (as was the case in the original code), passing a smaller buffer will alert any static analysis tools, as the code might want to access data beyond it.